baselines/fedprox/README.md

```diff
@@ -6,7 +6,6 @@ dataset: [MNIST]
 ---
```

Comment:
Above this line we have some metadata indicating the datasets used, could you add `FEMNIST`?